cli/flox/doc/flox-install.md

```diff
@@ -48,11 +56,10 @@ ID is set to that attribute.
 For pkg-paths that consist of multiple attributes (e.g. `python310Packages.pip`)
 the install ID is set to the last attribute in the pkg-path (e.g. `pip`).
 
-You may also specify packages to be installed via
-[`flox-edit(1)`](./flox-edit.md),
-which allows specifying a variety of options for package installation.
-See [`manfifest-toml(1)`](./manifest.toml.md) for more details on the available
-options.
+If a flake output attribute is specified instead of a pkg-path, the install ID
```

Comment:
```suggestion
If a Nix flake reference is specified instead of a pkg-path, the install ID
```
We haven't introduced what a flake is yet, so we need to clarify that it's not a concept native to Flox but to Nix.

Reply:
Changed to flake installable